src/config.rs

```diff
@@ -375,6 +401,25 @@ fn get_default_auth(ini: &Ini) -> Option<Auth> {
     }
 }
 
+fn get_default_model(ini: &Ini) -> Option<String> {
+    if let Some(ref val) = env::var("DEVICE_MODEL").ok() {
+        Some((*val).to_owned())
+    } else if let Some(val) = ini.get_from(Some("defaults"), "model") {
+        Some((*val).to_string())
+    } else {
+        None
+    }
+}
+
+fn get_default_family(ini: &Ini) -> String {
+    if let Some(ref val) = env::var("DEVICE_FAMILY").ok() {
+        val.to_owned()
+    } else if let Some(val) = ini.get_from(Some("defaults"), "family") {
+        val.to_string()
+    } else {
+        DEFAULT_FAMILY.to_owned()
```

Comment:
The above 2 cases work but not the constant `DEFAULT_FAMILY` here. I'm not sure why.